lib/presentation/widgets/word_cloud.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_scatter/flutter_scatter.dart';
+
+import '../../core/theme/app.dart';
+import '../models/word_cloud/word_cloud_data.dart';
+
+class WordCloudExample extends StatelessWidget {
```

Comment:
Write a unit test for that class. It will be located under `test/presentation/widget/word_cloud_test.dart`. You have a look at the unit test for `sentiment_chart_test.dart`

Reply:
Addressed in commit a8adfbf0e58487309908ff3f0f58fea5a6a01ab4